### PR TITLE
Tests: Tabs

### PR DESCRIPTION
**Pull request by amogower**
**What kind of change does this PR introduce?**

Added tests for Tabs component and sub-components.

**Does this PR introduce a breaking change?**

No.

### CODE REVIEW

**Comment by amogower** [comment]
Just have to look into this build error...

**Comment by PlayMa256** [comment]
weird error btw, i cannot find the root on TabsPanel. `tab` is being sent to the component everytime...

**Comment by amogower** [comment]
I found the issue, my VSCode linter didn't highlight an undefined var - my mistake.  Have pushed so should be sorted on next build.

File: packages/reakit/src/Tabs/__tests__/TabsPanel-test.tsx [hunk 1]
```diff
@@ -30,7 +30,8 @@ test("isCurrent", () => {
 
 test("tab", () => {
   const currentTab = "second";
-  const isCurrent = (idOrIndex: string | number): boolean => tab === currentTab;
```
Comment (matheus1lva) [inline packages/reakit/src/Tabs/__tests__/TabsPanel-test.tsx]: good catch!

And what a useful error message...

**Pull request merged**